catalog/app/containers/Auth/messages.js

```diff
@@ -0,0 +1,281 @@
+import { defineMessages } from 'react-intl';
+
+const scope = 'app.containers.Auth';
+
+export default defineMessages({
+  notificationAuthLost: {
+    id: `${scope}.notificationAuthLost`,
+    defaultMessage: 'Authentication lost. Please sign in again.',
```

Comment:
Same question, when does this happen? "~~Please~~Sign in again."

Reply:
this happens if a user is authenticated, but the next auth check receives 401 from the server (for example, if user's access was revoked, or if the token was invalidated for some reason)